app/blacklist.json

```diff
@@ -0,0 +1,10 @@
+{
+  "generator-express-angular": "haven't updated package.json",
+  "ft-wp": "haven't updated package.json",
+  "generator-angular-phonegap": "haven't updated package.json",
+  "generator-angular-js": "haven't updated package.json",
+  "generator-ionicjs": "duplicate entry of generator-ionic",
+  "generator-charcoal": "deprecated",
+  "generator-react-coffee-webpack": "duplicate entry of generator-react-webpack",
+  "generator-angular-rails-proxy": "didn't change their package.json (ಠ_ಠ)"
+}
```

Comment:
Or if they don't want to browse through pages of commits, they can just blame the specific line.